wrapper/Tools/OpenMMMD.py

```diff
@@ -703,6 +703,8 @@ def setupForcefields(system, space):
 
         for molnum in molnums:
             mol = molecules.molecule(molnum)[0].molecule()
+            if not mol.hasProperty("restrainedAtoms"):
+                continue
```

Comment:
Do you need the if above if you? Can't you just add an `except` to your try block to achieve the same result? Also, what should happen if there are no restrained atoms? Is `mol_restrained_atoms` undefined, or an empty list?